### PR TITLE
make SQL identifiers case insensitive (issue #45)

### PR DESCRIPTION
**Pull request by pkommoju**

### CODE REVIEW

**Comment by zhouqingqing** [comment]
@pkommoju  fix the codefactor issues please

File: qpmodel/Catalog.cs [hunk 1]
```diff
@@ -51,7 +51,11 @@ public class ColumnDef
         readonly public ColumnType type_;
         public int ordinal_;
 
-        public ColumnDef(string name, ColumnType type, int ord) { name_ = name; type_ = type; ordinal_ = ord; }
+        public ColumnDef(string name, ColumnType type, int ord)
+        {
+            name_ = name.StartsWith('"') ? name : name.ToLower();
```
Comment (zhouqingqing) [inline qpmodel/Catalog.cs:56]: can we add one utility function 
string normalizeName(string name) => name.StartsWith('"') ? name : name.ToLower();

to do the conversion?

Comment (pkommoju) [reply qpmodel/Catalog.cs:56]: Sure. Let me do it right away. Will adding it in Utils.cs work?

Comment (pkommoju) [reply qpmodel/Catalog.cs:56]: Looks like all source files use qpmodel.expr and expr.cs is used by all except PrestoStats.cs. So that should work.

**Comment by zhouqingqing** [comment]
can you fix the code factor issues?

**Pull request merged**